data/model/user_timeline_event.py

```diff
@@ -40,7 +41,13 @@ class RecordingRecommendationMetadata(pydantic.BaseModel):
     artist_msid: str
 
 
-UserTimelineEventMetadata = Union[RecordingRecommendationMetadata]
+class NotificationMetadata(pydantic.BaseModel):
+    creator_id: int
+    message: str
+    link: str
+
+
+UserTimelineEventMetadata = Union[RecordingRecommendationMetadata, NotificationMetadata]
 
 
 class UserTimelineEvent(pydantic.BaseModel):
```

Comment:
not directly related to this PR, but I see that in this class `created` is a datetime, whereas in APIFollowEvent it's an int. why is this?

Reply:
I do not know why that is. maybe @mayhem or @paramsingh know?

Reply:
Not a clue. We love our timestamp/datetime confusion in LB, that's for sure.